services/db/maintenanceModeHandler.go

```diff
@@ -26,7 +26,25 @@ func (handler *MaintenanceModeHandler) HasThresholdReached() bool {
 func (handler *MaintenanceModeHandler) Handle() {
 	logger.Info("Starting Maintenance Mode. Connecting to default DB 'postgres'")
 	dbname := config.GetEnv("JOBS_DB_DB_NAME", "ubuntu")
-	misc.ReplaceDB(dbname, "original_"+dbname+"_"+strconv.FormatInt(time.Now().Unix(), 10))
+	targetName := "original_" + dbname + "_" + strconv.FormatInt(time.Now().Unix(), 10)
+	misc.ReplaceDB(dbname, targetName)
+	loadDBConfig()
+	dbHandle := createDBConnection()
+	createWorkspaceTable(dbHandle)
+	workspaceData := getWorkspaceData(targetName)
```

Comment:
We can get workspace data before replacing the DB. Then we wouldn't need extra function  `loadOriginalDBConfig `